package.json

```diff
@@ -18,6 +18,7 @@
     "react-router-dom": "5.2.0",
     "react-router-guards": "1.0.0",
     "react-scripts": "4.0.3",
+    "socket.io-client": "4.1.3",
```

Comment:
Did you check version of ```socket.io-client``` in package-lock.json?

Reply:
In my package-lock.json: "socket.io-client": "^4.1.3"